app/routes/vet/sheep-eligibility.js

```diff
@@ -6,6 +6,8 @@ const session = require('../../session')
 const legendText = 'Will you have at least 21 sheep on the date of the review?'
```

Comment:
Can the text be updated to match the prototype please? https://sfi-prototype.herokuapp.com/ahw/vet/11/sheep/eligibility